gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/entities.proto

```diff
@@ -44,6 +44,7 @@ message EntitiesRequest {
 
   int32 limit = 20;
   int32 offset = 21;
+  bool include_results_outside_time_range = 22;
```

Comment:
We definitely need some naming experts here :) 
This doesn't seem to register that well but at the same time cant really suggest a better name either.

Reply:
How about `include_non_live_entities`?

Reply:
Yes. That sounds a lot better and clear.